packages/glimmer-compiler/lib/compiler.ts

```diff
@@ -19,7 +25,15 @@ export type TemplateSpec = string;
  * @return {TemplateSpec} A template spec string
  */
 export function compileSpec(string: string, options: any): TemplateSpec {
+  let meta: BlockMeta = {};
+  if (typeof options === 'object') {
+    meta = Object.assign(meta, options.meta);
```

Comment:
I don't think we can use `Object.assign` here, sadly.  It is not supported on IE9 - IE11, or on Node 0.10 - 0.12.